im.bernard.Nostalgia.json

```diff
@@ -10,6 +10,45 @@
         "--socket=wayland"
     ],
     "modules" : [
+        {
+            "name": "libsass",
+            "buildsystem": "meson",
+            "sources": [{
+                "type": "git",
+                "url": "https://github.com/lazka/libsass.git",
+                "branch": "meson"
```

Comment:
there should be a commit here

Reply:
actually branch should not be used, because branch change HEAD so the build would fail in that case.

commit: ok
tag + commit: ok (but if upstream retag BAD, it will fail)
branch: not ok

Reply:
Got it, thanks!